### DISCUSSION TITLE
APC inROW

### DISCUSSION THREAD

**Discussion by f0r7y** [netbox-community/netbox#10193] [Q&A] [answered]
Hi How can I manage APC inRow in Netbox?
Like this: https://www.apc.com/us/en/product/ACRD301S/inrow-dx-300mm-30kw/?range=61780-uniflair-direct-expansion-inrow-cooling&selected-node-id=27602434529

**Comment by jeremystretch** [comment]
I would model it as a non-racked device in the site (and optionally within a location).

(As an aside, 6ft tall, 1ft wide, 350 lbs, on casters? Sounds safe to me.)

**Comment by jcollie** [comment, marked as answer]
I model our Liebert in-row cooling systems by creating a rack for it and creating a device that fills the whole rack. Since these are narrower than a standard rack I don't think that they'll render quite right but at least they'll be there.

**Comment by f0r7y** [comment]
Thanks a lot
